examples/model_hosting/simple_train_predict/TrainAndPredict.ipynb

```diff
@@ -646,7 +653,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.5.0"
+   "version": "3.7.2"
```

Comment:
Isn't it better to use 3.5.0 if that's what GMLE is using?

Reply:
It is, but it will still work for 3.5. Most of the other notebooks are also marked for 3.7.